src/stores/ContractsStore.ts

```diff
@@ -1,48 +1,44 @@
 import { ContractsStore } from '@big-whale-labs/stores'
-import {
-  goerliDefaultProvider,
-  mainnetDefaultProvider,
-} from 'helpers/providers/defaultProvider'
-import {
-  goerliHeavyProvider,
-  mainnetHeavyProvider,
-} from 'helpers/providers/heavyProvider'
 import { proxy } from 'valtio'
 import { subscribeKey } from 'valtio/utils'
 import Network from 'models/Network'
 import WalletStore from 'stores/WalletStore'
 import env from 'helpers/env'
+import networks from 'networks'
+import transformObjectValues from 'helpers/transformObjectValues'
 
-export const GoerliContractsStore = proxy(
-  new ContractsStore(goerliDefaultProvider, goerliHeavyProvider, Network.Goerli)
-).makePersistent(env.VITE_ENCRYPT_KEY)
+const proxyNetworks = transformObjectValues(
+  networks,
+  ({ network, heavyProvider, defaultProvider }) =>
+    proxy(
+      new ContractsStore(defaultProvider, heavyProvider, network)
```

Comment:
Hm, maybe we can put default data inside `ContractStore`? If it's hard to do, let's leave as it is

Reply:
This class `ContractStore` is imported from the library